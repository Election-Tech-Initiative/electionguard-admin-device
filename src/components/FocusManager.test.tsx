import React from 'react'
import { render } from '../../test/testUtils'

import FocusManager from './FocusManager'

it('renders FocusManager', async () => {
  const { container } = render(<FocusManager>foo</FocusManager>)
  expect(container.firstChild).toMatchSnapshot()
})
