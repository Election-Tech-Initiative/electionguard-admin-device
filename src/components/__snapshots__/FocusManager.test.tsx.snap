// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FocusManager 1`] = `
<div
  class="sc-bdVaJa eMFrIq"
  tabindex="-1"
>
  foo
</div>
`;
